lib/saml/url_service.rb

```diff
@@ -182,5 +189,15 @@ def add_query(url, params)
         url
       end
     end
+
+    def create_redirect_application(uuid)
+      LoginRedirectApplication.create(
+        uuid: uuid, redirect_application: @redirect_application
+      )
+    end
+
+    def find_redirect_application(uuid)
```

Comment:
nitpick: a method called `find` doesn't' strike me a something that pops a record from redis.